cli/tools/registry/mod.rs

```diff
@@ -91,7 +91,7 @@ pub async fn publish(
 
   let cli_options = cli_factory.cli_options()?;
   let directory_path = cli_options.initial_cwd();
-  let publish_configs = cli_options.start_dir.jsr_packages_for_publish();
+  let mut publish_configs = cli_options.start_dir.jsr_packages_for_publish();
```

Comment:
I just realized this doesn't work when the deno.json doesn't have a version, but this is good enough for now.

Reply:
Oh wait, no it does because we consider deno.json files without a version as a package.